sys-auth/authenticator/authenticator-3.32.2.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+PYTHON_REQ_USE="sqlite"
+
+inherit gnome2-utils meson python-single-r1 virtualx xdg-utils
+
+DESCRIPTION="Two-factor authentication code generator for GNOME"
+HOMEPAGE="https://gitlab.gnome.org/World/Authenticator"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.gnome.org/World/Authenticator"
+else
+	SRC_URI="https://gitlab.gnome.org/World/Authenticator/-/archive/${PV}/Authenticator-${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+
+	S="${WORKDIR}/Authenticator-${PV}"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="gnome-shell test"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RESTRICT="!test? ( test )"
+
+CDEPEND="${PYTHON_DEPS}
+	app-crypt/libsecret[introspection]
+	>=gui-libs/libhandy-0.0.11:=[introspection]
```

Comment:
This doesn't seem to have a sublost specified, nor are there more than 1 version available in ::gentoo. `gui-libs/libhandy`